code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -40,7 +40,7 @@
 	icon_state = "deathsquad"
 	item_state = "swat_suit"
 	allowed = list(/obj/item/weapon/gun,/obj/item/ammo_box,/obj/item/ammo_casing,/obj/item/weapon/melee/baton,/obj/item/weapon/restraints/handcuffs,/obj/item/weapon/tank/internals)
-	armor = list(melee = 80, bullet = 80, laser = 50,energy = 50, bomb = 100, bio = 100, rad = 100)
+	armor = list(melee = 50, bullet = 40, laser = 30,energy = 50, bomb = 50, bio = 100, rad = 100)

```

Comment:
To be honest, this absolutely should not be merged until the Deathsquad changes are removed. This is an admin event, for when ERT cannot cut it. They are expected to deal with several threats at once. Inferior armor will break this.